tests/test_code_review.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import responses
+from unidiff import PatchSet
+
+from bugbug.tools.code_review import find_comment_scope
+
+
+def test_find_comment_scope():
+    responses.add_passthru("https://phabricator.services.mozilla.com/")
+    responses.add_passthru("https://d2mfgivbiy2fiw.cloudfront.net/file/data/")
```

Comment:
we might want to add mocks here, to ensure tests don't fail for intermittent reasons

Reply:
I agree; I filed a follow-up issue.